src/app/teams/teams.utils.ts

```diff
@@ -1 +1,10 @@
 export const memberCompare = (member1, member2) => member1.userId === member2.userId && member1.userPlanetCode === member2.userPlanetCode;
+
+export function compareFunc(member1, member2) {
+  const name1 = member1.userDoc.doc.lastName;
+  const name2 = member2.userDoc.doc.lastName;
+  if (name1 && name2) {
```

Comment:
This is a problem, because if one member has a name entered and the other does not it will sort both by id.  You should have `name1` equal `.lastName` if it exists and `.userId` if not and the same for `name2`.